lib/feature/pokemon_details_page.dart/widgets/pokemon_about_tab.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:pokedex_async_redux/api/models/ability_model.dart';
+import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/utilities/constants.dart';
+import 'package:pokedex_async_redux/utilities/extensions.dart';
+
+class AboutTab extends StatelessWidget {
+  const AboutTab({
+    required this.height,
+    required this.weight,
+    required this.abilities,
+    required this.baseExperience,
+  });
+  final int height;
+  final int weight;
+  final List<Ability>? abilities;
+  final int baseExperience;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.all(20),
+      child: Column(
+        children: [
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Row(
+              children: [
+                Padding(
+                  padding: EdgeInsets.fromLTRB(0, 0, 0, 5),
+                  child: Container(
+                    width: 150,
+                    child: Text(
+                      'Height',
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        color: aboutTextColor,
+                      ),
+                    ),
+                  ),
+                ),
+                Container(
+                  width: 60,
+                  child: Text('${height / 10} m'),
+                ),
+              ],
+            ),
+          ),
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Row(
+              children: [
+                Padding(
+                  padding: EdgeInsets.fromLTRB(0, 0, 0, 5),
+                  child: Container(
+                    width: 150,
+                    child: Text(
+                      'Weight',
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        color: aboutTextColor,
+                      ),
+                    ),
+                  ),
+                ),
+                Container(
+                  width: 60,
+                  child: Text('${weight / 10} kg'),
+                ),
+              ],
+            ),
+          ),
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Row(
+              children: [
+                Padding(
+                  padding: EdgeInsets.fromLTRB(0, 0, 0, 5),
+                  child: Container(
+                    width: 150,
+                    child: Text(
+                      'Abilities',
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        color: aboutTextColor,
+                      ),
+                    ),
+                  ),
+                ),
+                Container(
+                  width: 200,
+                  child:
+                      Wrap(children: abilities!.map((e) => Text('${(e.ability?.name ?? '').capitalize}, ')).toList()),
```

Comment:
null check instead of force unwrap

Reply:
this will result in an unwanted comm
![Screenshot 2022-07-28 at 4 51 44 PM](https://user-images.githubusercontent.com/47196695/181464484-b408700b-8cea-4ba9-9c14-bdcf9006460e.png)
a